src/pages/accountsettings/AccountSettings.js

```diff
@@ -0,0 +1,112 @@
+import React, { useState, Component } from "react";
+import { Link } from "react-router-dom";
+// import "../signup/signup.css";
```

Comment:
Put your styles in the signup.css and remove it from the common.css as mentioned before.